rpa-builder/6.2/modules/ROOT/pages/activity-library-explorer.adoc

```diff
@@ -0,0 +1,52 @@
+= Activity Library Explorer 
+
+In the Activity Library Explorer you can view all the activity templates assigned to your organization. All users in your organization with the privilege *Activity Library Administration* have access to the same Activity Library.
+
+If another user adds an Activity, you see the change after refreshing the Activity Library Explorer.
+
+image::activity-library-explorer.png["The Activity Librar Explorer window."]
+
+== Toolbar 
+
+The Activity Library Explorer toolbar contains the following elements: 
+
+* (image:activity-library-explorer-toolbar-refresh.png["The Refresh button.", 2%, 2%]) *Refresh*
++
+Updates the contents of the Activity Library Explorer.
+* (image:activity-library-explorer-toolbar-show-outline.png["The Show Outline button.", 2%, 2%]) *Show Outline*
++
+Shows an outline of the workflow.
+* (image:edit-icon.png["The Edit button.", 2%, 2%]) *Edit*
++
+Opens the individual editing mode of an activity to make changes.
++
+If you upload the activity after making changes, the Activity Library saves the updated activity as a new revision.
++
+You can edit the Activities in the Activity Library regardless of which phase the Process that contains these activities is in.
+* (image:activity-library-explorer-toolbar-show-log.png["The Show Log button.", 2%, 2%]) *Show Log*
++
+Opens a window containing all revisions of the activity. You can select a revision and edit it by choosing *Edit* in the individual editing mode.
```

Comment:
Kim prefers select over choose in this case. https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22select,%22%22choose%22

Reply:
Oh I see. You have select in the sentence already. Is *Edit* a button to click?